rotkehlchen/ethchain.py

```diff
@@ -53,17 +53,43 @@ def attempt_connect(self, ethrpc_port, mainnet_check=True) -> Tuple[bool, str]:
                         'the ethereum mainnet'.format(ethrpc_port)
                     )
                     return False, message
+                if self.web3.eth.syncing:  # pylint: disable=no-member
+                    curr_block = self.web3.eth.syncing.currentBlock  # pylint: disable=no-member
+                    high_block = self.web3.eth.syncing.highestBlock  # pylint: disable=no-member
+                    return self.is_syncronized(curr_block, high_block)
+                else:
+                    high_block = self.get_eth_highest_block()
+                    curr_block = self.web3.eth.blockNumber  # pylint: disable=no-member
+                    return self.is_syncronized(curr_block, high_block)
 
             self.connected = True
             return True, ''
         else:
-            logger.warn('Could not connect to a local ethereum node. Will use etherscan only')
+            logger.warning('Could not connect to a local ethereum node. Will use etherscan only')
             self.connected = False
             message = 'Failed to connect to ethereum node at port {}'.format(ethrpc_port)
 
         # If we get here we did not connnect
         return False, message
 
+    def is_syncronized(self, curr_block, high_block):
```

Comment:
Shouldn't it be `synchronized` here and in the other places?

Reply:
Yes. The `is_` prefix cues me to a boolean value but since it's actually returning a tuple[bool, str], the `is_` prefix would be incorrect anyways.

Reply:
ah no sorry.

I did not mean to remove `is_`. I meant that you wrote `is_syncronized` while I suppose it should be `is_synchronized`. It's missing an `h` , no?

Reply:
Oh haha! Yep, I see.